.github/workflows/ci.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Run golangci-lint
-      uses: golangci/golangci-lint-action@v2
+      uses: golangci/golangci-lint-action@v3
```

Comment:
My understanding when doing some research was that the `golangci-lint-action@v2` will try to install Go rather than using the already installed one in the previous step that uses `actions/setup-go@v2`. In this case the `golangci-lint-action@v2` seemed to be trying to pull Go 1.18 for linting. Using `golangci-lint-action@v3` allows us to use the Go installed in the previous setup making sure that we are using the same go version.